docs/getting-started/tutorial-load-dataset.asciidoc

```diff
@@ -78,7 +78,7 @@ curl -H 'Content-Type: application/json' -XPUT http://localhost:9200/shakespeare
 
 This mapping specifies the following qualities for the data set:
 
-* The _speaker_ field is a string that isn't analyzed. The string in this field is treated as a single unit, even if
+* The _speaker_ field is a field that isn't analyzed. The text in this field is treated as a single unit, even if
```

Comment:
I think I'd flip this around and say, Because the _speaker_ and _play_name_ fields are keyword fields, they are not analyzed. The strings are treated as a single unit even if they contain multiple words." and drop "The same applies to the _play_name_ field."